msu/systems/mod_settings/abstract_setting.nut

```diff
@@ -70,7 +76,7 @@
 
 	function reset()
 	{
-		this.set(this.BaseValue, true, true, true, true);
+		this.set(this.BaseValue, true, false, true, true);
```

Comment:
Can't do this because it changes existing behavior I think, maybe we need to aggregate these too.